Module2_HM1/Module2_HM1/Logger.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Module2_HM1
+{
+    public sealed class Logger
+    {
+        private static readonly Logger instance = new Logger();
+
+        private static readonly List<string> logs = new List<string>();
+        static Logger()
+        {
+        }
+
+        private Logger()
+        {
+        }
+
+        public static Logger Instance
+        {
+            get
+            {
+                return instance;
+            }
+        }
+
+        public void Log(LogType logType, string message)
+        {
+            logs.Add($"{DateTime.Now}: {logType}: {message} ");
```

Comment:
Probably a better way to store would be to have the actual Result stored in the List. That way you would be able: search, filter your logs